src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py

```diff
@@ -844,6 +860,7 @@ def _test_flexible_server_public_access_mgmt(self, database_engine, resource_gro
                  checks=NoneCheck())
 
 
+'''
```

Comment:
This test is intentionally commented out for few reasons. The goal of this test was to run the same commands with local context turned ON. However, the major problem this test was causing is :
1. The test should ideally run the commands and check for successful execution of the command **OR** it should validate the output of the commands. Currently, it is missing out on both. 
2. Since the tests execute in parallel and this test **turns on** the local context, it is inconsistently impacting the other tests caused by a race condition. If the local-context tests get kicked off before other tests, all commands in other tests is starting to pick up the values from local context since local context is turned on in the environment due to this  test. This leads to the failure of other live tests.

@DaeunYim will address these changes and send out a different PR.